grakn-dist/src/test/java/ai/grakn/dist/PhilosophersExampleIT.java

```diff
@@ -0,0 +1,74 @@
+package ai.grakn.dist;
+
+import ai.grakn.Grakn;
+import ai.grakn.GraknGraph;
+import ai.grakn.graql.MatchQuery;
+import ai.grakn.graql.QueryBuilder;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static ai.grakn.graql.Graql.var;
+import static java.util.stream.Collectors.joining;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PhilosophersExampleIT {
```

Comment:
Until we improve our functional test coverage or we auto run integration tests I do not think it wise to start marking tests as integration tests. These example tests catch MANY errors often missed by our functional tests. So either get IT tests auto running or change this back to a non-IT and mark it for future IT